packages/flutter/lib/src/material/search_anchor.dart

```diff
@@ -35,6 +35,7 @@ const Curve _kViewFadeOnInterval = Interval(0.0, 1/2);
 const Curve _kViewIconsFadeOnInterval = Interval(1/6, 2/6);
 const Curve _kViewDividerFadeOnInterval = Interval(0.0, 1/6);
 const Curve _kViewListFadeOnInterval = Interval(133 / _kOpenViewMilliseconds, 233 / _kOpenViewMilliseconds);
+const double _kDisableSearchBarOpacity = 0.38;
```

Comment:
Nit: Consider adding a comment explaining where you got this value. Did you eyeball it to look like the Material spec or something like that?

Reply:
Ah never mind, I see from [here](https://github.com/flutter/flutter/pull/137388#pullrequestreview-1778184310) that this value is hardcoded all over the codebase already. Maybe we should share 1 constant everywhere, but don't worry about it in this PR.